Dockerfile

```diff
@@ -9,11 +9,13 @@ COPY go.mod go.sum ./
 RUN go mod download
 COPY --chown=default:root exporters/**/*.go .
 
-RUN ["go", "build", "."]
+RUN CGO_ENABLED=0 GOOS=linux go build -o /bin/exporters .
 
 EXPOSE 8090
-CMD ["go", "run", "."]
 
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6-751
```

Comment:
I wonder what's the reasoning behind using ubi9-based image for the builder and ubi8-based for the final image. Can we use ubi9 for both?

Reply:
I tried using ubi9/ubi-minimal:9.2-750.1697625013 but it was stuck for a long time after installing some packages, so I used ubi8 for the second image. For the first image, Go Toolset for ubi9 contains the fix for CVE-2023-44487 so I have used ubi9 for first one.

Reply:
I think it's stuck because it's missing the `-y` on the install command.
It's waiting for user confirmation.